lucene/translate.py

```diff
@@ -154,36 +159,35 @@ def postprocess_translated_objc(path):
             if any(fnmatch.fnmatch(full_path_java, ptn) for ptn in excluded):
                 continue
 
-            full_path_m = full_path_java.replace(
-                src, dst).replace(".java", ".m")
+            full_path_m = full_path_java.replace(src, dst).replace(".java", ".m")
             if os.path.exists(full_path_m):
                 if os.path.getmtime(full_path_m) >= os.path.getmtime(full_path_java):  # nopep8
                     continue
             to_compile.append(full_path_java)
-            to_postprocess.append(full_path_m)
 
-    print('%s => %s files to compile' % (src, len(to_compile)))
+    print('\nProcessing: %s, %d java files to compile.\n' % (src, len(to_compile)))
 
-    if to_compile:
-        print('Compiling %d java files' % len(to_compile))
+    for file_to_compile in to_compile:
```

Comment:
This change (along with a later change that causes `args` to be invoked with each source will slow down the translation a lot. The reason I make `j2objc` translate the whole list of files is that J2ObjC will read all referenced classes and sources into memory and translate all the files mentioned in the (admittedly long) argument list in one go. With this change, the translator will be invoked again and again.